backend/YouTubeV2.Application/Services/SubscriptionsService.cs

```diff
@@ -28,5 +22,12 @@ public async Task<UserSubscriptionListDTO> GetSubscriptionsAsync(Guid Id, Cancel
                 Select(s => new SubscriptionDTO(new Guid(s.SubscribeeId), _blobImageService.GetProfilePicture(s.Subscribee.Id), s.Subscribee.UserName)).
                 ToListAsync(cancellationToken));
         }
+
+        public async Task<int> GetSubscriptionCount(Guid id, CancellationToken cancellationToken)
+        {
+            var userId = id.ToString();
+
+            return await _context.Subscriptions.Where(s => s.SubscribeeId == userId).CountAsync(cancellationToken);
```

Comment:
We should check if the userId is a id of a user that is a Creator also add Authorization to the endpoint we dont want anonymous users here. Idk if we can omit where and add lambda to Count but it is something to check

Reply:
I dont like it. This method will be called from user service not from a controller.  I think both authorization and creator check should be checked there not here. As for removing where - good suggestion.